contracts/validator_manager.v.py

```diff
@@ -106,16 +106,18 @@
 
 
 @public
-def __init__():
-    self.num_validators = 0
-    self.empty_slots_stack_top = 0
```

Comment:
Seems you just left them default initialized to `0`. Just curious is there any reference about the default initialization in vyper? In the document I only saw
> Variables initialized with the int128 type without an explicit value, such as nextFunderIndex, defaults to 0

in http://vyper.readthedocs.io/en/latest/vyper-by-example.html?highlight=default#crowdfund, but didn't see a more formal explanation.